src/main/java/core/basesyntax/service/serviceimpl/ProcessingServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.model.Activities;
+import core.basesyntax.model.ItemActivities;
+import core.basesyntax.service.ActivitiesService;
+import core.basesyntax.service.ProcessingService;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.fileservice.impl.FileReaderServiceImpl;
+import core.basesyntax.service.fileservice.impl.FileWriterServiceImpl;
+import core.basesyntax.service.strategy.ActivitiesHandler;
+import core.basesyntax.service.strategy.ActivitiesStrategy;
+import core.basesyntax.service.strategy.ActivitiesStrategyImpl;
+import java.util.List;
+import java.util.Map;
+
+public class ProcessingServiceImpl implements ProcessingService {
```

Comment:
Please move this logic to main method, there is no real sense here to create additional service to call all other services
You can use ProcessingService only for this method `processTransaction`

```
public interface ProcessingService {
    void processTransaction(List<ItemActivity> transactions);
}
```